File: 태성/가운데글자가져오기.py
s = "abcde"

def solution(s):
    answer = ''
    m = len(s)//2
    if len(s) % 2 == 1:
        answer = s[m]

    else:
        answer += s[m-1]
        answer += s[m]
    return answer

print(solution(s))

File: 태성/0615/가운데글자가져오기.js
const s = "abcde"	

function solution(s) {
  var answer = ''
  var m = parseInt(s.length/2)
  if (s.length % 2 === 1 ) {
    answer += s[m]
  } else {
    answer += s[m-1]
    answer += s[m]

  }
  return answer
}

console.log(solution(s))
